src/rdsn/src/common/backup.thrift

```diff
@@ -20,6 +20,30 @@ include "../../../../idl/dsn.layer2.thrift"
 
 namespace cpp dsn.replication
 
+enum backup_status
+{
+    INVALID,
+    CHECKPOINTING,
+    CHECKPOINTED,
```

Comment:
maybe CHECKING_POINT and CHECKED_POINT ?

Reply:
I think CHECKPOINT is a noun, the original is better.